src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC } from 'react';
+import { Todo } from '../../types/types';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+type Props = {
+  visibleTodos: Todo[];
+  tempTodo: Todo | null;
+  activeIds: number[];
+  handleRemoveTodo: (id: number) => void;
+};
+
+export const TodoList: FC<Props> = ({
+  visibleTodos,
+  tempTodo,
+  activeIds,
+  handleRemoveTodo,
+}) => (
+  <section className="todoapp__main">
```

Comment:
I suggest replacing `section` with `ul` tag for better HTML structure and styling.